src/wandbot/evaluation/eval/async_main.py

```diff
@@ -147,7 +153,7 @@ async def evaluate_row(idx: Hashable, row_str: str) -> str:
     eval_result.update(json.loads(await get_answer_correctness(row_str)))
     eval_result.update(json.loads(await get_answer_relevancy(row_str)))
     eval_result.update(json.loads(await get_answer_faithfulness(row_str)))
-
+    await asyncio.sleep(5)
```

Comment:
why sleep?

Reply:
Added it to prevent rate limiting at that time. Removed it now